templates/service.go

```diff
@@ -96,6 +108,10 @@ func (s *ServiceImpl) ListTokens(tType TokenType) (*[]BasicToken, error) {
 	return s.store.List(tType)
 }
 
+func (s *ServiceImpl) ListTokensFull(tType TokenType) ([]Token, error) {
```

Comment:
Why not return a pointer (`*[]Token`) like in `ListTokens` above?

Reply:
It doesn't make sense to return a pointer. The old code was dereferencing the pointer on all usages. Refactored here for conformity https://github.com/flow-hydraulics/flow-wallet-api/pull/314/commits/56ac3032c60f712976b29cd53e377717d9d9bdb8